### DISCUSSION TITLE
double click protection for submarine guns?

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#5619] [Ideas and suggestions]
Should there be a delay (hard coded or as a settable option) for triggering of periscope after user gets on the periscope?
inspiration:
Reed0506: yesterday my friend accidentally doubleclicked when he got on the railgun
Reed0506: i was then nugget
Reed0506: good time

**Comment by Reed0506** [comment]
yeah it was a moment for sure

**Comment by Ek-Videogames** [comment]
I often forget about the ramifications of default 'non-legacy' keybinds putting everything on mouse1.
This seems like a valid concern.

**Comment by thegrb93** [comment]
Whats double click do? I use legacy controls

**Comment by Reed0506** [comment]
double clicking is just one click to get on the periscope and one click to fire, which results in you immediately misfiring the railgun at wherever it was last aimed.
